packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -25,6 +25,10 @@ export class OpGroupingManager {
 		}
 
 		for (const message of batch.content) {
+			// Blob attaches cannot be grouped (grouped batching would hide metadata)
+			if (message.deserializedContent.type === ContainerMessageType.BlobAttach) {
+				return batch;
+			}
```

Comment:
I think this should just be deleted (not ported to next) since deserializedContent was deleted in `next` in #14462 @kian-thompson 

If that sounds good, I'll have the merge up shortly

Reply:
Yup, the change is just:
```
if (message.type === ContainerMessageType.BlobAttach) {
```